MATLAB/rl_ofdm_downlink_miso.m

```diff
@@ -414,11 +415,27 @@
 
 % Write beamformed signal to all antennas
 donwlink_postfix_len = N_SAMP - N_ZPAD_PRE - N_OFDM_SYMS * N_SYM_SAMP;
+tx_vec_iris_sampOffset = zeros(1,N_ZPAD_PRE + size(tx_payload_vec,2) + donwlink_postfix_len);
 for i=1:N_BS_NODE
     tx_signal = [zeros(1, N_ZPAD_PRE) tx_payload_vec(i, :) zeros(1, donwlink_postfix_len)];
     tx_vec_iris = TX_SCALE .* tx_signal ./ max(abs(tx_signal));
     % figure; plot(abs(tx_vec_iris))
-    node_bs.sdrtx_single(tx_vec_iris, i);       % Burn data to the UE RAM
```

Comment:
I have a hard time understanding the changes in this file. Does this improve upon the performance? I was not able to get past the `UPLINK PILOT COLLECTION: No LTS Correlation Peaks Found!` error when I tested it with the lab BS (it is all noise). The previous version where the downlink offset was randomly changing, could receive the uplink pilots with no issues.